docs/overview/peer-management.md

```diff
@@ -0,0 +1,17 @@
+# Peer Management
+![Peer Management](../assets/ldk-peer-management.svg)
+
+One of the first things you'll need to do when building your own lightning node is connect to a peer. In LDK this is handled by the [`PeerManager`](https://docs.rs/lightning/*/lightning/ln/peer_handler/struct.PeerManager.html).
+
+The `PeerManager` has two dependencies: 
+ - A [`ChannelManager`](https://docs.rs/lightning/*/lightning/ln/channelmanager/index.html) that enables it to notify peers when it wants to do things such as open channels.
+ - A [`P2PGossipSync`](https://docs.rs/lightning/*/lightning/routing/gossip/struct.P2PGossipSync.html) that understands when peers you are connected to send you gossip messages, enables you to forward messages, announce channel opens and more. 
+
+Since LDK knows nothing about your networking stack, you'll need to implement this module in order to connect to peers on the network. Provide LDK with raw TCP/IP socket data and the library will handle the rest for you.
```

Comment:
nit: A bit unclear what *this* refers to in this sentence. I believe the networking stack?